addons/l10n_in_ewaybill_stock/report/ewaybill_report.xml

```diff
@@ -220,29 +237,38 @@
                                                 (doc.env)).get(doc.mode, '')"/>
                                             </td>
-                                        <td>
-                                            <t t-if="doc.vehicle_no" t-out="doc.vehicle_no"/>
-                                            <t t-else="" t-out="doc.transportation_doc_no"/><t t-out="doc.transportation_doc_date"/>
-                                        </td>
-                                        <td>
-                                            <t t-out="generate_json['fromPlace']"/>
-                                        </td>
-                                        <td>
-                                            <t t-out="doc.ewaybill_date"/>
-                                        </td>
-                                        <td>
-                                            <t t-out="doc.company_id.vat"/>
-                                        </td>
+                                            <td>
+                                                <t t-if="doc.vehicle_no" t-out="doc.vehicle_no"/>
+                                                <t t-else="" t-out="doc.transportation_doc_no"/><t t-out="doc.transportation_doc_date"/>
+                                            </td>
+                                            <td>
+                                                <t t-out="generate_json['fromPlace']"/>
+                                            </td>
+                                            <t t-if="doc.state != 'pending'">
+                                                <td>
+                                                    <t t-out="doc.ewaybill_date"/>
+                                                </td>
+                                                <td>
+                                                    <t t-out="doc.company_id.vat"/>
+                                                </td>
+                                            </t>
+                                            <t t-else="">
+                                                <td>
+                                                    <t t-out="doc.distance"/>Km
+                                                </td>
+                                            </t>
                                         </tr>
                                     </tbody>
                                 </table>
                             </div>
                             <br/>
 
                             <div class="border-bottom border-dark text-center">
-                                <img t-if="doc.name != False" t-att-src="'/report/barcode/Code128/' + doc.name" style="width:250px;height:35px;" alt="Barcode"/>
-                                <br/>
-                                <t t-out="doc.name"/>
+                                <t t-if="doc.state != 'pending'" >
```

Comment:
Do we need this div (border) at the end of the page if it's empty?

Reply:
Hello @lordkrandel yes we need it everytime :)